R/get_-functions.R

```diff
@@ -310,8 +325,8 @@ get_duplicate_forecasts <- function(
   assert_data_frame(data)
   data <- ensure_data.table(data)
   forecast_unit <- get_forecast_unit(data)
-  available_type <- c("sample_id", "quantile_level") %in% colnames(data)
```

Comment:
Again if we can rely on the input being a forecast object we can make this s3 and avoid a new class requiring hard coded changes to the package

Reply:
Issue: https://github.com/epiforecasts/scoringutils/issues/888